c/headers/bpf/bpf_endian.h

```diff
@@ -0,0 +1,99 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
With the `nix` shell, there is no need to check in the headers, so those can be removed.